src/streaming/StreamProcessor.js

```diff
@@ -114,18 +113,14 @@ function StreamProcessor(config) {
             requestModifier: RequestModifier(context).getInstance()
         });
 
-        representationController = RepresentationController(context).create();
-        representationController.initialize(this);
-
         fragmentModel = scheduleController.getFragmentModel();
         fragmentModel.setLoader(fragmentLoader);
+
+        representationController = RepresentationController(context).create();
+        representationController.initialize(this);
     }
 
     function reset(errored) {
-        if (fragmentModel) {
-            fragmentModel.reset();
-            fragmentModel = null;
-        }
```

Comment:
Why does the `fragmentModel` no longer need to be reset or nulled out? It's still being setup in the `initialize` method.